### DISCUSSION TITLE
waiting in the shuffle

### DISCUSSION THREAD

**Discussion by GenOpenDiesel** [ORelio/Minecraft-Console-Client#2091] [Q&A] [answered]
Hi, is it possible to make the task to execute the command "/login password" 30 seconds after entering the server

**Comment by BruceChenQAQ** [comment]
You can use Script Scheduler / Task Scheduler

doc: https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config#using-the-script-scheduler--task-scheduler

https://github.com/MCCTeam/Minecraft-Console-Client/blob/master/MinecraftClient/config/sample-tasks.ini

**Comment by milutinke** [comment, marked as answer]
1. Open `MinecraftClient.ini` in Notepad or some text editor
2. Find `[ScriptScheduler]` using CTRL + F (or whatever is the key binding for search in your editor)
3. Set `enabled` to `true`
4. Set `tasksfile` to `tasks.ini`

It should be like this:
```ini
[ScriptScheduler]
# Schedule commands and scripts to launch on various events such as server join, date/time or time interval
# See README > 'Using the Script Scheduler' for more info
enabled=true
tasksfile=tasks.ini
```

4. Create a `tasks.ini` file.
5. Open it up and add this:
```ini
[Task]
triggerOnLogin=true
script=on-login.txt
```
6. Save it and close.

7. Create a `on-login.txt`
8. Add the following:
```
wait 30
send /login yourpasswordhere
```
9. Save and close.
10. Fire up the MCC
